submissions/benjaminKamena/html/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Contact page</title>
+</head>
+<body>
+    <div>
+        <h2>Contact Us</h2>
+        <p>Swing by for more products, or leave us a message:</p>
+    </div>
+    <div>
+        <form>
+            <label for="fullname">Full Name</label><br>
```

Comment:
@kamena1994 Well done. I think you should write this label in `fullName`  instead of `fullname`. Change it everywhere else you have written  that.